syntax = "proto3";

package authentication.grpc_config;

service AuthService {
    rpc Authenticate (AuthRequest) returns (AuthResponse);
}

message AuthRequest {
    string token = 1;
}

message AuthResponse {
    bool valid = 1;
    string user_id = 2;
}
